app/models/manageiq/providers/nuage/network_manager/event_catcher/runner.rb

```diff
@@ -22,6 +23,15 @@ def monitor_events
     stop_event_monitor
   end
 
+  def do_work_loop
+    super if has_amqp
+  end
+
+  def start_event_monitor
+    tid = super if has_amqp
+    return tid unless tid.nil?
+  end
```

Comment:
From what I understand we override the `start_event_monitor` function here so that when `has_amqp` is `false`, nothing is started, yes? If this is true, then I think no other change in this file is required, right? I mean, 

- do we then need to modify L3?
- do we then need to add L17?
- do we then need to also override `do_work_loop` in L26?

I may be missing something, but perhaps we can simplify this PR without affecting the functionality.

Reply:
Yes, it's a WIP. I remember doing this before the holiday season, and there were still some log outputs if I only prevented starting the event monitor. I'll write some tests and do more testing in development env.